roles/ceph-osd/tasks/activate_osds.yml

```diff
@@ -3,7 +3,8 @@
 # partition.
 
 - name: activate osd(s) when device is a disk
-  command: ceph-disk activate "{{ item | regex_replace('^(\/dev\/cciss\/c[0-9]{1}d[0-9]{1})$', '\1p') | regex_replace('^(\/dev\/loop[0-9]{1})$', '\1p') }}1"
+  command: ceph-disk activate "{{ item | regex_replace('^(\/dev\/cciss\/c[0-9]{1}d[0-9]{1})$', '\1p') | regex_replace('^(\/dev\/loop[0-9]{1})$', '\1p') }}
+                               {%- if 'nvme' in item %}{{ 'p1' }}{% else %}{{ '1' }}{% endif %}"
```

Comment:
Why mixing different approaches to address the same issue?

Reply:
I would keep using the filters
```yaml
{{ item | regex_replace('^(\/dev\/nvme.*)$', '\1p') }}1
```

Reply:
@guits, what you mean? Please explain. To use separated tasks - one for ATA and second for NVMe?

Reply:
@ommoreno I think we could use the same way you used for all the cases, remove all regex and only use jinja templating. It is more readable, what do you think?

Reply:
@guits I agree, that works for me.